influxql/average.go

```diff
@@ -0,0 +1,20 @@
+package influxql
+
+func averageFloat(wt, lt, nt int64, prev, next float64) float64 {
```

Comment:
Added comments and changed code format so things should be more clear now. Let me know if theres anything else I can add.